manifests/istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -68,8 +68,14 @@ data:
     {{- else }}
     mixerReportServer: istio-telemetry.{{ .Values.global.telemetryNamespace }}.svc.{{ .Values.global.proxy.clusterDomain }}:9091
     {{- end }}
+    {{- else }}
+    {{- if and .Values.telemetry.v1.tcp.enabled .Values.telemetry.enabled }}
```

Comment:
Can we add this condition to the preceding clause?

Reply:
Removed as we are not adding new TCP specific value